common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -66,6 +66,26 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-contributions-banner-environment-moment-supporters",
+    "new moment lorem ipsum",
```

Comment:
will the 'lorem ipsum' text be removed before go-live?

Reply:
👍 fixed